org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/BaseDocumentLifeCycleHandler.java

```diff
@@ -703,7 +703,6 @@ private void refreshLocalResource(IResource resource, int depth, IProgressMonito
 					workspace.beginOperation(true);
 					build = file.getLocalManager().refresh(file, IResource.DEPTH_ZERO, true, monitor);
 				} catch (OperationCanceledException e) {
-					workspace.getWorkManager().operationCanceled();
```

Comment:
I've removed this line because of this change: https://github.com/eclipse-platform/eclipse.platform/pull/478

I'm not sure if there is a better way to adapt to this change, but it seems like we don't always set the operation to canceled elsewhere when the `OperationCanceledException` is caught.